### PR TITLE
Add highlighting for diff on tabbankaccount

### LINKED ISSUE #326: Bank Reconcilliation highlight Reconciled banktransactions - bank vouchers
When you press "lukket" or "Diff(xxx,xx)" you should highlight the ones it is reconciled with.
Something like what is done on post mot post.

### PR DESCRIPTION
**Pull request by agvald**
Closes #326

### CODE REVIEW

File: modules/bank/model/bank.class.php [hunk 1]
```diff
@@ -497,6 +497,8 @@ public function getDiff($AccountPlanID, $KID, $InvoiceID, $JournalID, $TotalAmou
         if(!$value) {
             $value = $this->get_vouchertilbake($AccountPlanID, $KID, $InvoiceID, $JournalID);
         }
+        $value = ($value * (($From == "voucher" && $InnOrOut == "inn") ? -1 : 1));
```
Comment (agvald) [inline modules/bank/model/bank.class.php:502]: comment why

File: modules/bank/model/bank.class.php [hunk 2]
```diff
@@ -481,14 +481,14 @@ public function is_closeable($AccountPlanID, $KID, $InvoiceID, $JournalID) {
...
-    public function getDiff($AccountPlanID, $KID, $InvoiceID, $JournalID, $TotalAmount) {
+    public function getDiff($AccountPlanID, $KID, $InvoiceID, $JournalID, $TotalAmount, $InnOrOut, $From = "") {
         $KID        = trim($KID);
         $InvoiceID  = trim($InvoiceID);
         $JournalID   = trim($JournalID);
         $value      = 0;
 
         if(empty($JournalID) || (empty($KID) && empty($InvoiceID))) {
-          return $TotalAmount;
+          return ($TotalAmount * (($From == "voucher") ? -1 : 1));
```
Comment (agvald) [inline modules/bank/model/bank.class.php:492]: comment why

File: modules/bank/view/tabbankaccount.php [hunk 3]
```diff
@@ -174,10 +179,25 @@ function display_kontoliste($conf) {
...
+        setTimeout(function() {
+          $(elementid).removeClass("highlighted");
+        } , 5000);
+      }
+    });
     </script>
 </head>
 <body>
 
+<?function create_span($value, $ID){ return "<span class=\"navigate to\" id=$ID>$value</span>";}?>
```
Comment (agvald) [inline modules/bank/view/tabbankaccount.php]: add description for the function and make it in multiple lines

**Review by mladjo2505 — APPROVED** [review]
All good, ready for final review

**Comment by mladjo2505** [comment]
First round review: Mladen Drobnjaković (@mladjo2505)
Final review with: @stoivo

**Review by stoivo — APPROVED** [review]

**Pull request merged**